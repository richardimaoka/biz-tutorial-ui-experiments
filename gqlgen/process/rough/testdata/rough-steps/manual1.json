{
  "phase": "サクサク感を表現するためgowは必須",
  "type": "commit",
  "instruction": "add createdAt to Todo",
  "commit": "c8a922d8850d7b5f738f649a8ad61947f1202edd",
  "comment": ""
}
