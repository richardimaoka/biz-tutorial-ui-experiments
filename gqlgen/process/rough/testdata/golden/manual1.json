[
  {
    "step": "",
    "focusColumn": "Source Code",
    "commit": "c8a922d8850d7b5f738f649a8ad61947f1202edd",
    "defaultOpenFilePath": "data/todo.json"
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "data/todo.json",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "go.mod",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "go.sum",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "gqlgen.yml",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "graph/generated.go",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "graph/model/models_gen.go",
    "isFoldFileTree": true
  }
]